src/views/AQLWizard.vue

```diff
@@ -71,36 +113,6 @@
           >
             Run Query
           </v-btn>
-
-          <v-menu
-            v-model="createTableMenu"
-            offset-y
-            :close-on-content-click="false"
-          >
-            <template #activator="{ on }">
-              <v-btn
-                class="ml-2"
-                color="secondary"
-                v-on="on"
-              >
-                Create Table
-              </v-btn>
-            </template>
-            <v-card class="px-2 pt-2">
-              <v-text-field
-                v-model="createTableName"
-                solo
-                flat
-                label="Table Name"
-                append-icon="publish"
-                :error-messages="createTableErrorMessage"
-                :hide-details="!createTableErrorMessage"
-                @input="createTableErrorMessage = null"
-                @click:append="createTable"
-                @keydown.enter="createTable"
-              />
-            </v-card>
-          </v-menu>
```

Comment:
Why remove this? It's a cool feature of this view

Reply:
It doesn't actually work (there is no API endpoint that allows creation of a table from an AQL query), and adding it is out of scope for this PR. 

I don't know when the feature was removed / lost from the API, it obviously used to work though.

Reply:
Gotcha. Makes sense. Happy to merge as is then